File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/19_compito_java_20180129/Residenziale.py
from Cliente import Cliente


class Residenziale(Cliente):
    def __init__(self, tipo, codice, nome_cognome, tipo_contratto, indirizzo):
        super().__init__(tipo, codice, tipo_contratto, indirizzo)
        self._nome_cognome = nome_cognome
        
    def get_nome(self):
        return self._nome_cognome

    def __str__(self):
        return super().__str__() + "\t" + self._nome_cognome + "\t-\t" + self._indirizzo + "\t" + self._tipo_contratto + "\t-"

File: Programming-languages-and-laboratory/Java/Exercises/19_compito_java_2018-01-29/Residenziale.java
import java.util.*;

public class Residenziale extends Cliente {

	private String nome;

	public Residenziale(int id, String indirizzo, Boolean tipo_contratto, String nome) {
		super(id, indirizzo, tipo_contratto);
		this.nome = nome;
	}

	public String getNome() {
		return nome;
	}

	public String toString() {
		return id + "\tresidenziale\t" + nome + "\t-\t" + super.toString() + "-";
	}

}